src/natupnp_v2.erl

```diff
@@ -312,24 +328,14 @@ status_info(#nat_upnp{service_url=Url}) ->
 
 %% internals
 
-get_location(Raw) ->
-    case erlang:decode_packet(httph_bin, Raw, []) of
-        {ok, {http_error, _}, Rest} ->
-            get_location(Rest);
-        {ok, {http_header, _, 'Location', _, Location}, _Rest} ->
-            Location;
-        {ok, {http_header, _, _H, _, _V}, Rest} ->
-            get_location(Rest);
-        _ ->
-            error
-    end.
-
 get_service_url(RootUrl) ->
     case httpc:request(RootUrl) of
         {ok, {{_, 200, _}, _, Body}} ->
             {Xml, _} = xmerl_scan:string(Body, [{space, normalize}]),
             [Device | _] = xmerl_xpath:string("//device", Xml),
             case device_type(Device) of
+                "urn:schemas-upnp-org:device:InternetGatewayDevice:1" ->
```

Comment:
The service was answering with a version 1 instead of 2.

Reply:
👍